src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -49,12 +50,34 @@ object VertexSchema {
   }
 }
 
+object EdgeSchema {
+  val sep = "_"
+  sealed trait EdgeRelationType {
+    def prefix: String
+    def toLabel(label:String) = prefix + sep + label
+  }
+  case object Own extends EdgeRelationType {
+    override val prefix = "OWN"
+  }
+  case object Ref extends EdgeRelationType {
+    override val prefix = "REF"
+  }
+  val allEdgeRelations = Seq(Own, Ref)
+
+  def getEdgeRelation(label: String): EdgeRelationType = {
+    toEdgeRelation(label.split(sep).head)
+  }
+
+  def toEdgeRelation(prefix: String): EdgeRelationType = {
+    allEdgeRelations.filter( er => prefix == er.prefix ).head
+  }
```

Comment:
Yeah, I suppose it is. I guess what would be nice is the ability to serialize classes into edge labels, so that any arbitrary information can be encoded there ... but that way leads to madness. So, direct string manipulation is fine.